plugin/trino-exchange/src/test/java/io/trino/plugin/exchange/FileSystemExchangeQueryRunner.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.exchange;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.log.Logger;
+import io.airlift.log.Logging;
+import io.trino.Session;
+import io.trino.plugin.tpch.TpchPlugin;
+import io.trino.testing.DistributedQueryRunner;
+
+import static io.airlift.configuration.ConfigurationLoader.loadPropertiesFrom;
+import static io.trino.testing.TestingSession.testSessionBuilder;
+
+public class FileSystemExchangeQueryRunner
```

Comment:
not used

Reply:
Currently this is used as a convenient utility for me to start a local server using s3 exchange spooling and test locally. Depending on what kind of tests we are going to create I will either keep it or delete it.